### PR TITLE
Fix variables that upgrade 009-custom didn't fixed

### PR DESCRIPTION
**Pull request by Stell0**
https://github.com/nethesis/dev/issues/6007

### CODE REVIEW

File: root/usr/share/tancredi/scripts/upgrade.d/011-custom.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+foreach ($custom_fixes as $model_id => $variables) {
+    $scope = new \Tancredi\Entity\Scope($model_id, $container['storage'], $container['logger']);
+    if(isset($scope->metadata['version']) && $scope->metadata['version'] >= 11) {
+        continue;
+    }
+    $scope->metadata['version'] = 11;
+    foreach ($variables as $variable) {
+        $scope->setVariables($variable);
+        $container['logger']->info("Fix ".basename(__FILE__)." applied to scope $model_id: ".array_keys($variable)[0]." => ".array_values($variable)[0]);
```
Comment (DavidePrincipi) [inline root/usr/share/tancredi/scripts/upgrade.d/011-custom.php:52]: I prefer to write a fixed message string: it's easier to search it in log files and source code and it's also more readable in the source code.

For instance:

```suggestion
        $container['logger']->info("Fix 011-custom.php: set fanvil-X3U fanvil_lkpages_count=0");
```

**Review by DavidePrincipi — COMMENTED** [review]
Why not using a fixed log string?

**Review by DavidePrincipi — COMMENTED** [review]
Do we have to comment it?


https://github.com/nethesis/nethserver-tancredi/blob/fb164217353eed6bf06139288980f4542fbbf762/root/usr/share/tancredi/scripts/upgrade.d/009-custom.php#L28

**Comment by Stell0** [comment]
I'd avoid to change it... it is broken, but already released and at least everything is broken in the same way

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**